Assets/MRTK/SDK/Features/UX/Scripts/Pointers/SpherePointer.cs

```diff
@@ -189,6 +189,49 @@ public bool TryGetNearGraspPoint(out Vector3 result)
             }
         }
 
+        /// <summary>
+        /// Gets the position of where grasp happens
+        /// For IMixedRealityHand it's the axis from the palm to the grasp point
+        /// For any other IMixedRealityController, return just the pointer's forward orientation
+        /// </summary>
+        public bool TryGetNearGraspAxis(out Vector3 result)
+        {
+            using (TryGetNearGraspPointPerfMarker.Auto())
```

Comment:
We probably want a new perf marker for this new method, to differentiate form the grasp point perf marker